apps/src/lib/levelbuilder/lesson-editor/UploadImageForm.jsx

```diff
@@ -26,18 +27,22 @@ export default function UploadImageForm() {
 
     // assemble upload data
     const formData = new FormData();
-    formData.append('file', e.target.files[0]);
-    setFormDataForImage(formData);
-    setTempImageUrl(URL.createObjectURL(e.target.files[0]));
+    let tempImageUrlList = [];
+    for (let i = 0; i < e.target.files.length; i++) {
+      formData.append('file', e.target.files[i]);
+      tempImageUrlList.push(URL.createObjectURL(e.target.files[i]));
+    }
+    setFormDataForImages(formData);
```

Comment:
Right now we are storing a list of image files in a FormData object stored in component state (`[formDataForImages, setFormDataForImages]`), but we actually don't use this object as FormData, we parse it as a list and create sub-form datas from it. Given how its used, all else being equal, I would prefer that this was stored as an array of File (`File[]`) rather than as form data that coincidentally only has Files as members.

Reply:
Want to add that if you didn't feel like changing this, I would not look askance at all.